Detectors/FIT/FT0/calibration/testWorkflow/FT0ChannelCalibration-Workflow.cxx

```diff
@@ -0,0 +1,58 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "FT0Calibration/FT0CalibrationInfoObject.h"
+#include "FITCalibration/FITCalibrationDevice.h"
+#include "FT0Calibration/FT0ChannelDataTimeSlotContainer.h"
+#include "FT0Calibration/FT0CalibrationObject.h"
+
+using namespace o2::fit;
+
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  workflowOptions.emplace_back(o2::framework::ConfigParamSpec{"initialTimestamp",
+                                                           o2::framework::VariantType::Int, 0,
+                                                           {"Timestamp of initial calibration object that will be read from CCDB"}});
+}
+
+#include "Framework/runDataProcessing.h"
+
+using namespace o2::framework;
+WorkflowSpec defineDataProcessing(ConfigContext const& config)
+{
+
+  using CalibrationDeviceType = o2::fit::FITCalibrationDevice<o2::ft0::FT0CalibrationInfoObject,
+    o2::ft0::FT0ChannelDataTimeSlotContainer, o2::ft0::FT0CalibrationObject>;
+
+
+  auto initialTimestamp = config.options().get<int>("initialTimestamp");
```

Comment:
I don't see why do you need this `initialTimestamp` and how it can be accommodated in the sync. processing? 
I understand that you need the "starting" object, but for every TF the framework is supposed to provide to you the freshest available object. Is not this enough?

Reply:
No,why, at every TF the framework will provide you most up to date object.

Reply:
But it is not implemented yet, right?.

Reply:
Not yet, and the test objects in the http://ccdb-test.cern.ch:8080/browse/?report=true  are currently loaded with 1 year validity. So, if you put there something and read with timestamp "now", you will get most up-to-date object. Right now you can even keep this `initialTimestamp`, but in the real production the framework will take care of shipping the fresh object.

Reply:
"So, if you put there something and read with timestamp "now", you will get most up-to-date object." 
I though that I have to provide the exact (initial)timestamp of the object to retrieve it.
Thanks! I did not know about it. So right, this value can be easily removed.